sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java

```diff
@@ -0,0 +1,20 @@
+package io.sentry.android.core;
+
+import android.content.Context;
+import io.sentry.core.Sentry;
+import io.sentry.core.util.NonNull;
+
+/** Sentry initialization class */
+public final class SentryAndroid {
```

Comment:
IIRC @mitsuhiko had some thoughts WRT introducing some second `init` like this, maybe reagarding docs or whatnot.
I'd just like to get his review here before we move forward.
/cc @HazAT